src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -55,6 +55,7 @@ protected function configure(): void
                 new InputOption('show-aliases', null, InputOption::VALUE_NONE, 'Show aliases in overview'),
                 new InputOption('format', null, InputOption::VALUE_REQUIRED, \sprintf('The output format ("%s")', implode('", "', $this->getAvailableFormatOptions())), 'txt'),
                 new InputOption('raw', null, InputOption::VALUE_NONE, 'To output raw route(s)'),
+                new InputOption('method', null, InputOption::VALUE_REQUIRED, 'Set the HTTP method', null),
```

Comment:
```suggestion
                new InputOption('method', null, InputOption::VALUE_REQUIRED, 'Set the HTTP method', null, ['GET', 'POST', 'PUT', 'DELETE', 'PATCH']),
```
To help autocompletion?

(I don't think that we should suggest `CONNECT`, `TRACE`, `HEAD`, etc)

Reply:
I will definitely make that change, thanks for the suggestions.